contents/blog/posthog-vs-mixpanel.md

```diff
@@ -0,0 +1,925 @@
+    <thead>
+        <tr>
+			<td className="w-3/12"></td>
+            <td><strong>Mixpanel</strong></td>
+            <td><strong>PostHog</strong></td>
+            <td></td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><strong>Feature flags</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Roll out features safely; toggle features for cohorts or individuals</td>
+        </tr>
+        <tr>
+            <td><strong>Session recording</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Watch users interact with your product; diagnose bugs</td>
+        </tr>
+        <tr>
+            <td><strong>Experiments</strong></td>
+            <td className="text-center">Enterprise</td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Test changes in production before rolling them out</td>
+        </tr>
+        <tr>
+            <td><strong>Heatmaps</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Track where users click and why</td>
+        </tr>
+        <tr>
+            <td><strong>In-app guides</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Create in-app feature guides</td>
+        </tr>
+        <tr>
+            <td><strong>NPS & surveys</strong></td>
+			  <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-orange text-lg">⚠</span></td>
+            <td>Create, report on surveys, gather feedback</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+- **Experiments:** Mixpanel offers an experiments feature in its Enterprise plan, but it's limited to running simple A/B tests. Users on Mixpanel's Growth plan can integrate with third-party testing tools like AB Tasty or Convert and analyze results in Mixpanel. PostHog's experimentation suite supports A/B and multivariate tests using our integrated feature flagging.
+
+- **Feature flagging:** Mixpanel doesn't have any built-in feature flag functionality, but there are official integrations for Flagsmith, GrowthBook, and others. See: [What are the benefits of an all-in-one platform?](anchor-link-here)
+
+- **NPS & surveys:** PostHog doesn't have a built-in survey feature, but it's easy to [create your own surveys and send the results to PostHog for analysis](/tutorials/survey).
```

Comment:
We have the feedback app: https://github.com/PostHog/feedback-app

Reply:
@joethreepwood Is this one of the ones you're doing an app page for at the moment?